js_modules/dagit/packages/core/src/launchpad/LaunchpadAllowedRoot.tsx

```diff
@@ -55,6 +74,20 @@ export const LaunchpadAllowedRoot = (props: Props) => {
   const pipelineOrError = result?.data?.pipelineOrError;
   const partitionSetsOrError = result?.data?.partitionSetsOrError;
 
+  const filteredRootDefaultYaml = React.useMemo(() => {
+    if (result.data?.runConfigSchemaOrError.__typename !== 'RunConfigSchema') {
```

Comment:
Could do `const runConfigSchemaOrError = result.data?.runConfigSchemaOrError` outside the `useMemo` so you can use that in the memo instead of repeating `result.data.runConfigSchemaOrError`.